linera-sdk/src/views/system_api.rs

```diff
@@ -9,23 +9,23 @@ use async_trait::async_trait;
 use linera_base::ensure;
 use linera_views::{
     batch::{Batch, WriteOperation},
-    common::{ContextFromDb, KeyValueStoreClient},
+    common::{ContextFromStore, KeyValueStore},
     views::ViewError,
 };
 
 /// We need to have a maximum key size that handles all possible underlying
 /// sizes. The constraint so far is DynamoDb which has a key length of 1024.
 /// That key length is decreased by 4 due to the use of a value splitting.
-/// Then the KeyValueStoreClient needs to handle some base_key and so we
+/// Then the KeyValueStore needs to handle some base_key and so we
 /// reduce to 900. Depending on the size, the error can occur in system_api
 /// or in the KeyValueStoreView.
 const MAX_KEY_SIZE: usize = 900;
 
 /// A type to interface with the key value storage provided to applications.
 #[derive(Default, Clone)]
-pub struct KeyValueStore;
+pub struct WasmStore;
```

Comment:
(Or maybe `AppStateStore` or `ContractStateStore`?)

Reply:
`SystemStore` or `Application(State)Store` work for me.

(`ContractState` is too narrow. `Wasm` is not the right concept I think)